src/Libraries/DesignScriptBuiltin/Builtin.cs

```diff
@@ -29,6 +30,17 @@ public static object ValueAtIndex(IList list, int index)
                 }
                 return list[index];
             }
+
+            public static object ValueAtIndex(string stringList, int index)
+            {
+                while (index < 0)
```

Comment:
what is the purpose of this loop?

Reply:
Indexing from the end of the list with negative indices.